pkgs/build-support/setup-hooks/wrap-gapps-hook.sh

```diff
@@ -2,13 +2,43 @@
 gappsWrapperArgs=()
 
 find_gio_modules() {
-    if [ -d "$1/lib/gio/modules" ] && [ -n "$(ls -A "$1/lib/gio/modules")" ] ; then
+    if [ -d "$1"/lib/gio/modules ] && [ -n "$(ls -A $1/lib/gio/modules)" ] ; then
```

Comment:
Why drop the quoting?

Reply:
Hmm, I think it could have either been `shellcheck` or `shellfmt` that did this.

Reply:
and wouldn't `"$(ls -A "$1/lib/gio/modules")"` be too much quoting?

Reply:
nope
```
In pkgs/build-support/setup-hooks/wrap-gapps-hook.sh line 5:
    if [ -d "$1/lib/gio/modules" ] && [ -n "$(ls -A $1/lib/gio/modules)" ] ; then
                                                    ^-- SC2086: Double quote to prevent globbing and word splitting.
```

weird, will fix.